css/team.css

```diff
@@ -134,9 +134,7 @@ h3 {
 
 /* Current positions */
 .current-positions {
-	width: 90%;
```

Comment:
added wrapper on several areas as requested and deleted width where it was conflicting. thank you